src/routes/users/deleteUser.ts

```diff
@@ -52,11 +56,25 @@ const schema = {
     tags: ["User"],
     summary: "Delete a User",
     params: UserRequestParamsSchema,
-    description: "Endpoint for deleting a user.",
+    description: "Endpoint for deleting an user.",
     response: {
         204:{},
-        404:ErrorSchemaJson,
-        500:ErrorSchemaJson
+        404: {
+          title: "InvalidUser",
+          description: "Invalid or missing User.",
+          type: "object",
+          properties: {
+            error: ErrorSchemaJson,
+          },
+        },
+        500: {
```

Comment:
Remove this status code, please. Given that we are returning the error object in the catch we don't need this one.